editions/tw5.com/tiddlers/widgets/CheckboxWidget.tid

```diff
@@ -18,9 +18,10 @@ The content of the `<$checkbox>` widget is displayed within an HTML `<label>` el
 
 |!Attribute |!Description |
 |tiddler |Title of the tiddler to manipulate (defaults to the [[current tiddler|Current Tiddler]]) |
-|tag |The name of the tag to which the checkbox should be bound |
+|tag |The name of the tag to which the checkbox is bound |
 |invertTag |When set to ''yes'', flips the tag binding logic so that the absence of the tag causes the checkbox to be checked  |
-|field |The name of the field to which the checkbox should be bound |
+|field |The name of the field to which the checkbox is bound |
+|index|<<.from-version "5.1.14">> The index of the //tiddler//, a [[DataTiddler|DataTiddlers]], to which the checkbox is bound<br>» be sure to set the //tiddler// correctly |
```

Comment:
Haven't paid much attention to documentation macros. Using the .tip macro looks good enough to me.